recipes/mender-client-version-inventory-script/debian-master/rules

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/make -f
+
+export MENDER_VERSION := $(shell python3 -c "import json; print(json.load(open('versions.json'))['mender'])")
```

Comment:
This is dangerous. The version must come from the debian recipe, we cannot rely on the contents of the json file. We _could_ check that it is the same (with an exception for experimental builds), but not take it as the source of truth.